src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -28,6 +28,7 @@ String user = (String) request.getSession().getAttribute("user");
 <head>
   <title><%= conversation.getTitle() %></title>
   <link rel="stylesheet" href="/css/main.css" type="text/css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Can you explain what this is? Is it a font style?

Reply:
This is for the gear icon